devtools/ci/travis/install.sh

```diff
@@ -35,13 +35,13 @@ else # Otherwise, CPython... go through conda
         conda install -y -n myenv boost==1.59.0 -c omnia
         conda install -y -n myenv nglview -c bioconda
         conda install -y ambertools=17.0.0 -c http://ambermd.org/downloads/ambertools-rc/conda/
-        python -c "import sander; print(sander)"
-        python -c "import parmed; print(parmed)"
     else
         # Do not install the full numpy/scipy stack
         conda create -y -n myenv python=$PYTHON_VERSION numpy nose pyflakes=1.0.0 \
             coverage nose-timer python-coveralls
     fi
 
     source activate myenv
+    python -c "import sander; print(sander)"
+    python -c "import parmed; print(parmed)"
```

Comment:
uhm super weird, the ambertools conda works fine for this repo:

https://api.travis-ci.org/jobs/222133069/log.txt?deansi=true (search for ambertools)

```bash

The following NEW packages will be INSTALLED:

    ambertools: 17.0-0 http://ambermd.org/downloads/ambertools-rc/conda

ambertools-17. 100% || Time: 0:00:03  34.51 MB/s

$ tleap -h
-I: Adding /home/travis/miniconda2/dat/leap/prep to search path.
-I: Adding /home/travis/miniconda2/dat/leap/lib to search path.
-I: Adding /home/travis/miniconda2/dat/leap/parm to search path.
-I: Adding /home/travis/miniconda2/dat/leap/cmd to search path.
Usage: /home/travis/miniconda2/bin/teLeap [options]
Options:
 -h         Generate this message.
 -s         Ignore leaprc startup file.
 -I {dir}   Add {dir} to search path.
 -f {file}  Source {file}.


The command "tleap -h" exited with 0.
$ sqm -h
sqm [-O] -i <input> -o <output>
   -O           Overwrite output file if it exists
   -i <input>   Input file
   -o <output>  Output file
   -h           Show this message


The command "sqm -h" exited with 0.
$ cpptraj -h

Usage: cpptraj [-p <Top0>] [-i <Input0>] [-y <trajin>] [-x <trajout>]
               [-c <reference>] [-d <datain>] [-w <dataout>] [-o <output>]
               [-h | --help] [-V | --version] [--defines] [-debug <#>]
               [--interactive] [--log <logfile>] [-tl]
               [-ms <mask>] [-mr <mask>] [--mask <mask>] [--resmask <mask>]
       cpptraj <Top> <Input>
	-p <Top0>        : Load <Top0> as a topology file. May be specified more than once.
	-i <Input0>      : Read input from <Input0>. May be specified more than once.
	-y <trajin>      : Read from trajectory file <trajin>; same as input 'trajin <trajin>'.
	-x <trajout>     : Write trajectory file <trajout>; same as input 'trajout <trajout>'.
	-c <reference>   : Read <reference> as reference coordinates; same as input 'reference <reference>'.
	-d <datain>      : Read data in from file <datain> ('readdata <datain>').
	-w <dataout>     : Write data from <datain> as file <dataout> ('writedata <dataout>).
	-o <output>      : Write CPPTRAJ STDOUT output to file <output>.
	-h | --help      : Print command line help and exit.
	-V | --version   : Print version and exit.
	--defines        : Print compiler defines and exit.
	-debug <#>       : Set global debug level to <#>; same as input 'debug <#>'.
	--interactive    : Force interactive mode.
	--log <logfile>  : Record commands to <logfile> (interactive mode only). Default is 'cpptraj.log'.
	-tl              : Print length of trajectories specified with '-y' to STDOUT.
	-ms <mask>       : Print selected atom numbers to STDOUT.
	-mr <mask>       : Print selected residue numbers to STDOUT.
	--mask <mask>    : Print detailed atom selection to STDOUT.
	--resmask <mask> : Print detailed residue selection to STDOUT.



The command "cpptraj -h" exited with 0.
$ python -c "import parmed; print(parmed)"
<module 'parmed' from '/home/travis/miniconda2/lib/python2.7/site-packages/parmed/__init__.py'>


The command "python -c "import parmed; print(parmed)"" exited with 0.
$ python -c "import sander; print(sander)"
<module 'sander' from '/home/travis/miniconda2/lib/python2.7/site-packages/sander/__init__.py'>


The command "python -c "import sander; print(sander)"" exited with 0.
$ python -c "import pytraj; print(pytraj)"
<module 'pytraj' from '/home/travis/miniconda2/lib/python2.7/site-packages/pytraj/__init__.py'>


The command "python -c "import pytraj; print(pytraj)"" exited with 0.
$ python -c "import nglview; print(nglview)"
<module 'nglview' from '/home/travis/miniconda2/lib/python2.7/site-packages/nglview/__init__.pyc'>


The command "python -c "import nglview; print(nglview)"" exited with 0.

Done. Your build exited with 0.
```